lab-jaren/lib/game-router.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Game = require('../model/game.js');
+const storage = require('../lib/storage.js');
+
+module.exports = function(router) {
+  router.post('/api/games', function(req, res) {
+    if(!req.body.title || !req.body.genre || !req.body.developer || !req.body.publisher || !req.body.platforms || !req.body.ratingESRB || !req.body.releaseDate) {
```

Comment:
Some more refactoring can happen here. Your instantiation of game can be moved up after Line 7 and this code still works in place of that gnarly if statement.

```
let game = new Game(req.body);
    for (let key in game){
      if(game[key] === undefined){
        res.statusCode = 400;
        res.end();
        return;
      }
    }
```